### DISCUSSION TITLE
Bluescsi v2 in dos/Windows 95

### DISCUSSION THREAD

**Discussion by HarryNorway** [BlueSCSI/BlueSCSI-v2#89] [Help]
I have connected a BlueSCSI v2 to a windows 95 PC. I prepared a 2 GB img file with the name **HD60_512 games.img** in dosboxX and loaded it onto a SD card.

the img. file shows up in dos and windows 95 showing the files I put on the virtual harddrive. But when I try to copy the files to my C: drive, I get an error that windows can't read the files from the virtual drive. If I try to copy a folder, the folder gets copied, but the  the files within gives a read error.

If I try to copy from scsi virtual D: drive to C: in Dos, I don't get any errors and it looks like its copying, but no files are showing up in the destination folder.

Anyone got any ideas about what the problem could be?

**Comment by erichelgeson** [comment]
Sorry for the late reply - I'm unfortunately not very familiar with Windows and SCSI. I'd start with the trouble shooting guide in the wiki (don't skip any steps!) and see if that helps resolve the issue. Could be a power or termination issue if you're getting gibberish. Maybe a debug log after you go through the trouble shooting guide would provide some details as well.

**Comment by Orzipan** [comment]
Maybe try with a < 512 mb disk image?

**Reply by HarryNorway** [reply]
I got it working. Using Windows 11 I made a 2 gb VHD in windows manager and formatted the drive in FAT16 using Commandprompt (CMD). This made it read and write in Bluescsi, using dos, windows 3.11 and Windows 95. 

This is what I used I CMD:
"By entering "format X: /FS:FAT/Q" at your command prompt, where X: is the drive letter of your USB flash drive, you may format the disk to FAT16."
